app/views/courses/CoursesView.coffee

```diff
@@ -231,3 +238,9 @@ module.exports = class CoursesView extends RootView
     courseInstanceID = $(e.target).data('courseinstance-id')
     window.tracker?.trackEvent 'Students View Levels', category: 'Students', courseID: courseID, courseInstanceID: courseInstanceID, ['Mixpanel']
     application.router.navigate("/students/#{courseID}/#{courseInstanceID}", { trigger: true })
+
+  onClickViewProjectGalleryLink: (e) ->
+    courseID = $(e.target).data('course-id')
+    courseInstanceID = $(e.target).data('courseinstance-id')
+    window.tracker?.trackEvent 'Students View Project Gallery', category: 'Students', courseID: courseID, courseInstanceID: courseInstanceID, ['Mixpanel']
```

Comment:
What's the benefit? Having the event in multiple places is redundant and not future proof (if there's another link added pointing to the project gallery page, one has to somehow remember to add this specific event).